.readthedocs.yaml

```
@@ -0,0 +1,14 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# RTD API version
+version: 2
+
+build:
+  os: ubuntu-lts-latest
+  tools:
+    python: >-
+      3.11
+  commands:
+  - mkdir -pv "${READTHEDOCS_OUTPUT}"/html/
+  - cp -rv docs/* "${READTHEDOCS_OUTPUT}"/html/
```

By the way, if you drop slashes like this, the manual dir creation wouldn't be needed:
```suggestion
  - cp -rv docs "${READTHEDOCS_OUTPUT}"/html
```

---

@webknjaz I get this:

```
[rtd-command-info] start-time: 2024-06-04T14:17:29.321762Z, end-time: 2024-06-04T14:17:29.384388Z, duration: 0, exit-code: 1
cp -rv docs "${READTHEDOCS_OUTPUT}"/html
cp: cannot create directory '/home/docs/checkouts/readthedocs.org/user_builds/oranod-ansible-docs-deploy/checkouts/latest/_readthedocs//html': No such file or directory
```

---

Oh.. I see! That must be because the parent directory structure isn't there. Makes sense, then. Marking as resolved.